build-scripts/build-custom-app.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# Define the content for the main.tsx file
+read -r -d '' MAIN_CONTENT <<EOM
+
+import React, { FC, ReactNode } from "react";
+import Header from "@components/headers/Main";
+import Footer from "@components/footer/Footer";
+
+type TProps = {
+  children?: ReactNode;
+};
+
+const Main: FC<TProps> = ({ children }) => (
+  <>
+    <Header />
+    <main id="main" className="flex flex-col flex-1">
+      {children}
+    </main>
+    <Footer />
+  </>
+);
+export default Main;
+
+EOM
+
+# Define the content for Analytics.tsx
+read -r -d '' ANALYTICS_CONTENT <<EOM
+
+import React from 'react';
+
+const Analytics = ({enableAnalytics}: {enableAnalytics: boolean}) => {
+    return null 
+};
+
+export default Analytics;
+
+EOM
+
+# Define the content for methodology.tsx
+read -r -d '' METHODOLOGY_PAGE_CONTENT <<EOM
+
+const Methodology = () => {
+  return null;
+};
+
+export default Methodology;
+
+EOM
+
+# Define the content for homepage.tsx
+read -r -d '' HOMEPAGE_CONTENT <<EOM
+
+const LandingPage = ({}) => <h1> Hello World </h1>;
+
+export default LandingPage;
+
+
+EOM
+
+# Define the content for the routes.json
+
+read -r -d '' ROUTES_JSON_CONTENT <<EOM
+
+[
+  {
+    "title": "Methodology",
+    "path": "methodology",
+    "contentPath": "methodology"
+  }
+]
+
+EOM
+
+# Prompt the user for the name of the new directory
+read -r -p "Enter the name of the new custom app (use camel-case if required): " custom_app_name
```

Comment:
Do we want the user to input the full name of the app/customer, or the acronym we refer to them by?

Reply:
@katybaulch umm i am going to have to lean on you and what you think is best here